packages/block-library/src/template-part/edit/index.js

```diff
@@ -150,6 +151,9 @@ export default function TemplatePartEdit( {
 				  )
 				: false;
 
+			const _canEditTemplate =
+				select( coreStore ).canUser( 'create', 'templates' ) ?? false;
+
```

Comment:
Is this check correct? Should it check if a user can edit templates or template parts?

Reply:
@Mamaduka As far as I know they use one singular REST Controler for both. And that lives under `templates` 🤔 

I may of course have missed something

Reply:
They have different rest bases - `templates` and `template-parts`.

I'm asking because I'm working to bulk update `canUser` selectors, and I would instead use the correct post type for new checks. See #63415.

Reply:
I've updated to the correct entity in #63415.